javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,60 @@
+let selects = document.querySelectorAll('select');
+let fromSelect = document.querySelector("#from-currency");
+let toSelect = document.querySelector("#to-currency");
+let inputAmount = document.querySelector("#amount");
+let convertedAmount = document.querySelector("#convertedAmount");
+
+
+// Fetch data from the API
+async function fetchData(urlAPI) {
```

Comment:
It could just take the currency code as parameter and have the api url inside the funtcion as that part stayes the same